### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eirenik0**
After cloning dirname is pdf.js not pdfjs and need to install npm dependecies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,12 @@ The version of the extension for the Opera browser can be found at the [Opera ad
 To get a local copy of the current code, clone it using git:
 
     $ git clone git://github.com/mozilla/pdf.js.git pdfjs
-    $ cd pdfjs
+    $ cd pdf.js
```
Comment (Snuffleupagus) [inline README.md]: If you execute the *actual* `git clone` command above, which explicitly specifies the directory name, the folder will be named `pdfjs`. Please revert this change.

File: README.md [hunk 2]
```diff
@@ -52,11 +52,12 @@ The version of the extension for the Opera browser can be found at the [Opera ad
...
 
 Next, you need to start a local web server as some browsers don't allow opening
 PDF files for a file:// url:
-
+
```
Comment (Snuffleupagus) [inline README.md]: Nit: please remove the trailing whitespaces added here.

File: README.md (hunk 1 above)
Comment (CodingFabian) [inline README.md]: or maybe change the actual clone command to use pdf.js ? then its the same url you would get as when using github tools to clone.

Comment (timvandermeij) [inline README.md]: That seems fine to me too. I always use just `git clone git://github.com/mozilla/pdf.js.git` personally and therefore my working folder has always been `pdf.js`. It would make the command somewhat easier and I can't think of any problems. Objections, other ideas anyone?

**Comment by timvandermeij** [comment]
Fixed with some added improvements in #5991. Thanks!
